### PR TITLE
Add soft-credit list resources

### PR DESCRIPTION
**Pull request by dpfavand**
Adding "soft-credits" list resources to campaigns, fundraising teams, and fundraising pages. Needed for a current project.

### CODE REVIEW

File: src/resources.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
             "questions",
             "registrations",
             "recurring-donation-plans",
+            "soft-credits"
```
Comment (mjalkio) [inline src/resources.json]: Missing a comma here.

**Comment by dpfavand** [comment]
@mjalkio - missing comma added. Could you reconsider for merging? Thanks.

**Comment by mjalkio** [comment]
Hi @dpfavand, thanks for the contribution :) it looks good to me!

However, I'm not the primary maintainer of this repo. Pinging @mlingner to see if he's taken a look at this. I'll follow up with him tomorrow during business hours.

**Comment by mlingner** [comment]
Merged and published in latest version on npm. Thanks @dpfavand for the contribution.

**Pull request merged**